core/new-gui/src/app/workspace/component/left-panel/operator-menu/operator-label/operator-label.component.spec.ts

```diff
@@ -53,15 +53,6 @@ describe("OperatorLabelComponent", () => {
     expect(component).toBeTruthy();
   });
 
-  it("should generate an ID for the component DOM element", () => {
-    expect(component.operatorLabelID).toContain("texera-operator-label-");
-  });
-
-  it("should display operator user friendly name on the UI", () => {
-    const element = <HTMLElement>fixture.debugElement.query(By.css(".texera-operator-label-body")).nativeElement;
-    expect(element.firstChild?.textContent?.trim()).toEqual(mockOperatorData.additionalMetadata.userFriendlyName);
-  });
-
```

Comment:
can we modify the test cases instead of removing them?

Reply:
Those test cases are poorly designed. It's not a good idea to test the dom element by CSS or the order of the dom element as they are changing frequently.

Reply:
The 2nd test case is to check if we display user friendly name for operators. If it is poorly designed, can we suggest a good design?

Reply:
We removed the first one because we no longer generate an ID for the component DOM element.
The second one is fixed.